src/App.tsx

```diff
@@ -1,61 +1,41 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { PostForm } from './components/PostForm/PostForm';
+import { getUser } from './services/user';
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+function getNewTodoId(todoList: Todo[]) {
+  const maxId = Math.max(...todoList.map(todo => todo.id));
+
+  return maxId + 1;
+}
+
+export const App: React.FC = () => {
+  const [todosList, setTodosList] = useState<Todo[]>(todos);
+
+  const addTodo = ({ id, ...data }: Todo) => {
+    const newTodo = {
+      id: id || getNewTodoId(todosList),
```

Comment:
You are using `getNewTodoId(todosList)` to create a new id for the added Todo. This approach might lead to problems in case if ids are not contiguous. A safer approach could be using a unique id generator library like `uuid`.